### DISCUSSION TITLE
Release schedule?

### DISCUSSION THREAD

**Discussion by gwennlbh** [mirrorkeydev/Appstract#25] [Q&A] [answered]
Is there a schedule/frequency at which new versions are released? I'm really hyped about getting the new icons on my phone, but I don't know how android app publication works (stuff like a limit on the # of releases per month, delays between submitting a new version and getting it to land, etc) or how often does @mirrorkeydev have the time to publish new versions

TL;DR: How often do releases on the play store happen?

**Comment by mirrorkeydev** [comment, marked as answer]
Hi @ewen-lbh! In the past, when I created icons by myself, I usually published every 3-6 months, but now that there are outside contributors, I can see how that would be a bit too long of a time :) I can't publish too frequently because publishing icons does take fair chunk out of my day; between coordinating the two apps, fixing build errors, and personally testing the new release on my own device, it usually takes around 3 hours. So my goal is **not to go longer than 1.5 months without publishing a new release** (as long as there have been new icons contributed in that time)! Since I last released November 15-17th (rollouts can take a couple days), that means I'm due for a next release around 12/30/2020 - 1/1/2021

Hope that answers your question!
